### DISCUSSION TITLE
Construction time lapse 9 Month PROJ

### DISCUSSION THREAD

**Discussion by amatsumedia** [gopro/labs#775] [Q&A]
GoPro 11 with Labs Construction issue
Hey Everyone!
I have purchased a GoPro 11 and have installed Labs on it. I have used the
https://gopro.github.io/labs/control/construction/
I did start at 7:00
End at 17:00
and 145 Pics per day
---------------------------------------------
This is the one I made
https://gopro.github.io/labs/control/set/? cmd=mPdP>06:59<17:00!233SQ~!07:00S!1R&title=Daily%20Timelapse
---------------------------------------------

I have connected it to a Power Supply..
So far it looks like it's only doing it every 3 days ?
it shot 32 pics on FEB 8 but started late
it shot 103 pics on FEB 11
Then 3 pics on feb 14

Not sure what's going on .. Any ideas here?
Would Also like to have an auto Upload after each pic is taken if that works? I wouldn't mind if device stayed on the entire time either.. It seems as though it turns of after every pic...

this could be the issue

Need to get this going desperately! Please help!

**Comment by dnewman-gpsw** [comment]
View the MISC/qrlog.txt file (on your SD card) to learn more what is going on.  

You did see "Deprecated - This style is less reliable than the above types **Daily Long Time-lapses**"?

This construction style time lapse is not considered reliable, it was deprecated about a year ago. If you have external power, you could just start a regular timelapse, and if your power is reliable, it will still be recording months later. I've shot for three months this way. The Labs timelapse commands are more optimized for extending the camera battery to its limits. So the construction style works okay under battery alone, just need to swap the battery every few days. The script below is a little more power hungry, but more reliable than what you are using. Try https://gopro.github.io/labs/control/dailytl/  It will produce a timelapse video for each day, with one upload per day.    

"auto Upload after each pic is taken if that works?" With a further decrease in reliability. You shouldn't do that.  Labs allows to you try, and the scripting language does support this.

**Reply by amatsumedia** [reply]
So no way to really do Timelapse via images? Was hoping to get som better quality clips ( looks like that is better achieved by getting images - then potentially bulk editing via LR then importing as sequence )  If not I'll just attempt this method. Problem is this is being shot 15 stories up pointing down to a foundation across the street... A bit far

**Reply by dnewman-gpsw** [reply]
You absolutely can do photos, there is a time-lapse photo option in camera, no Labs required. You haven't explained why you need to power down the camera between photos, as it seems you have external power, all the scripts are for projects running on battery.

**Reply by amatsumedia** [reply]
Interesting .. Okay.. So Is there a way to have a timer set so that Timelapse's start and stop a certain time everyday within the UI of the GoPro 11 or with this just have to be a continuous option... ? 

My primary objective is to for example capture an image every 20 minutes every Mon - Friday  From 7 AM to 5 PM
If I Have to do every day of the week that's Fine as well.. But biggest thing I'm trying to accomplish is the stop and start time. 

Again Thanks for all your help.. I'm just new to the Timelapse / GoPro world...

**Reply by dnewman-gpsw** [reply]
You can do that, still you haven't explained why, other than just avoid having unneeded photos at night or the weekend.  If you want to most reliable, have it shot continuously. This is a 9 month project.

This next most reliable has to leave the camera on during the day.  Use Night Lapse Photo, as it has 20s interval, regular timelapse does not.  Night Lapse is fine for the day too. You have to switch the night lapse preset to Photo.

Then the simplest script is just  ```>17:00!07:00NmNPp.20eA!S!17:00E!1R```   This is shoots 7am to 5pm everyday.

Using Blocky you can customize to shoot Mon through Fri.   https://gopro.github.io/labs/build/

![image](https://github.com/gopro/labs/assets/10856084/a0cf2b7f-cb7f-400a-9655-9f29c1ccf3e5)

Use this image to further edit you script.
![Daily TimeLapse-2_19_2024, 09_50_28](https://github.com/gopro/labs/assets/10856084/0ba90153-f6b0-462e-bef3-053a5b044659)
